src/mapml/layers/DebugLayer.js

```diff
@@ -28,7 +28,7 @@ export var DebugOverlay = L.Layer.extend({
     this._grid = debugGrid({
       className: "mapml-debug-grid",
       pane: map._panes.mapPane,
-      zIndex: 10000,
+      zIndex: 400,
```

Comment:
I found that `400` is the lowest `z-index` value that seemingly doesn't visibly break anything _and_ fixes the issue with borders overlapping the tooltip.